doc_user/UserManual.adoc

```diff
@@ -491,6 +491,7 @@ The following functions are available from the *Buttons panel* of the window *_S
 
 * *C[.underline]##h##arge*: record a new charge movement.
 * *[.underline]##D##ischarge*: record a new discharge movement.
+* *[.underline]##D##elete*: delete the last movement.
```

Comment:
The underlined character should be the one used as _Mnemonic_ in the code and cannot be shared with other buttons in the same window.

According with the current code in develop now, the Delete button has been mapped with common `angal.common.delete.btn.key=D`, but in this windows we might map differently because we already have `angal.medicalstock.discharge.btn.key=D`, so we could define new `angal.medicalstock.discharge.btn=Delete` and `angal.medicalstock.delete.btn.key=L` or having `angal.medicalstock.discharge.btn.key=R`.

Since we did the same for Charge and Close button (keeping the "C" for common operation "Close") I would assume the best choice is to keep "D" for common operation of "Delete" and having Charge=H and Discharge=R.

In any case this doc should be updated accordingly.

Reply:
please should I apply those changes now so that after you will create the task to update it on gui ?

Reply:
we are still in the matter of OP-1020, just create a new branch from the latest develop and open a new PR for it

Reply:
> I would assume the best choice is to keep "D" for common operation of "Delete" and having Charge=H and Discharge=R.

this

Reply:
Okay the task is to just change the  key of charge and discharge.